src/config/index.js

```diff
@@ -1,3 +1,6 @@
+// const apiDomain = process.env.REACT_APP_API_DOMAIN.replace('\'', '');
+const apiDomain = 'https://unifree-backend.herokuapp.com';
+
```

Comment:
If I recall, correctly you are doing this because of some error your are getting. can you add some comments on what the error is?

Reply:
I resolve the error, was setting the wrong .env file, so process.env doesn't work